packages/web/src/components/dog-ear/DogEar.module.css

```diff
@@ -1,61 +1,49 @@
-.artistPick {
```

Comment:
I don't know why this class was named `artistPick` (probably the dogEar was originally just for that?), but it was clearly the "container" 😄